src/lib/paddles.ts

```diff
@@ -94,6 +94,22 @@ function useMachineTypes() {
   return useQuery(["machine_types", { url }]);
 }
 
+function useNodes(params: GetURLParams): UseQueryResult<Node[]> {
+  const params_ = JSON.parse(JSON.stringify(params || {}));
+
+  const queryString = new URLSearchParams(params_).toString();
+  let uri = "nodes" + (queryString? `?${queryString}` : '');
```

Comment:
```suggestion
  let uri = "nodes/" + (queryString? `?${queryString}` : '');
```
`/nodes/` isn't loading on the PR preview, and I see '302 Found' coming back from paddles. Maybe this'll fix that.